packages/experiments-realm/productivity/task.gts

```diff
@@ -276,51 +276,69 @@ function shortenId(id: string): string {
 }
 
 class Fitted extends Component<typeof Task> {
+  get visibleTags() {
+    if (!this.args.model.tags) return [];
+    return this.args.model.tags.slice(0, 2);
+  }
+
+  get hasMoreTags() {
+    if (!this.args.model.tags) return false;
+    return this.args.model.tags.length > 2;
+  }
+
+  get remainingTagsCount() {
+    if (!this.args.model.tags) return '';
+    return this.args.model.tags.length - 2;
+  }
+
   <template>
     <div class='task-card'>
-      <div>
-        <div class='main-info'>
-          <span class='short-id'>{{@model.shortId}}</span>
-          <h3 class='task-title'>{{@model.taskName}}</h3>
-        </div>
+      <div class='main-info'>
+        <span class='short-id'>{{@model.shortId}}</span>
+        <h3 class='task-title'>{{@model.taskName}}</h3>
+      </div>
 
-        <div class='sub-info'>
-          {{#if @model.assignee}}
-            <div class='task-assignee'>
-              <Avatar
-                class='avatar'
-                @userId={{@model.assignee.id}}
-                @displayName={{@model.assignee.name}}
-                @isReady={{true}}
-              />
-              <span class='assignee-name'>
-                {{@model.assignee.name}}
-              </span>
-            </div>
-          {{/if}}
+      <div class='sub-info'>
+        {{#if @model.assignee}}
+          <div class='task-assignee'>
+            <Avatar
+              class='avatar'
+              @userId={{@model.assignee.id}}
+              @displayName={{@model.assignee.name}}
+              @isReady={{true}}
+            />
+            <span class='assignee-name'>
+              {{@model.assignee.name}}
+            </span>
+          </div>
+        {{/if}}
 
-          <span class='task-date-range'>
-            <Calendar width='14px' height='14px' class='calendar-icon' />
-            {{#if @model.dueDate}}
-              <span class='date-info'><@fields.dueDate /></span>
-            {{else}}
-              <span class='date-info'>No Due Date Assigned</span>
+        {{#if @model.tags.length}}
+          <div class='task-tags'>
+            {{#each this.visibleTags as |tag|}}
+              <Pill class='tag-pill' @pillBackgroundColor={{tag.color}}>
```

Comment:
If you don't want to change the Pill component, the other option is to add `style={{cssVar
                  pill-background-color=tag.color
                  pill-font-color=(getContrastColor tag.color)
                }}` directly here. However, I'm also fine with the change to the Pill component if we expect to use this a lot.